pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
```

Comment:
The thing I just learned that is missing from the `setuptools_scm` documentation for some reason, is that removing the version here means needing to add `dynamic = ["version"]` to the project metadata.

Thanks for doing this btw